jepsen/scalardb/README.md

```diff
@@ -9,43 +9,42 @@ The current Scalar DB tests work with [Cassandra test tools in Jepsen](https://g
 1. Get Jepsen which has Cassandra tests
```

Comment:
Can you update the first section ?
```
# Jepsen tests for Scalar DB

This guide will teach you how to run Jepsen tests for Scalar DB.
The current tests use [Cassandra test tools in Jepsen](https://github.com/scalar-labs/jepsen/tree/cassandra).
```
@iamcap7  Please correct my english as needed.

Reply:
@feeblefakie It is not needed 👍

Reply:
Thanks. Done